doc/constrain_R14_trend.md

```diff
@@ -0,0 +1,67 @@
+## Contrain neuton stars Equation of State (EoS) 
+
+### create an output 
+
+First of all, you need to creat an output directory, this output will host all the data that will be used to constrain the EoS.
+    
+	mkdir -p ./nmma/output
+
+
+### gennerate EoS injection.json
+
+Run this command line will genarate  a json file (injection.json)  with the BILBY processing of compact binary merging events (BNS, NSBH, optional), here it's BNS. This injection contents a simulation of the parameters : luminosity_distance, log10_mej_wind, KNphi, inclination_EM, KNtimeshift, geocent_time.
+At the end of the run, when the process is ok,  an injection.json will be find at  ./nmma/output directory.
+
+	nmma_create_injection --prior-file ./nmma/example_files/prior/ZTF_kn.prior --eos-file ./nmma/example_files/eos/ALF2.dat --binary-type BNS -f ./nmma/output/injection --n-injection 100 --original-parameters  --extension json
+
+
+### create the direction of the EMdata 
+
+EMdata will house the posteriors of Electromagnetic data,  the lc.csv (./nmma/example_files/csv_lightcurve/outdir/macroeventID, where macroeventID in range(0, 100) )lightcurves.
+
+	mkdir -p ./nmma/output/EMdata
+
+### lightcurve posterior 
+
+The next command line provide some EM posterios in this example we have a simulation of a sample of 100 events and and 26 of them are detectable by the ZTF. 
+
+	for macroeventID in {0..99}
+
+	do
+	 light_curve_analysis --model Bu2019lm --svd-path ./nmma/svdmodels --gptype tensorflow  --outdir ./nmma/output/EMdata/outdir/$macroeventID --label injection_Bu2019lm --prior ./nmma/example_files/prior/ZTF_kn.prior --tmin 0 --tmax 7 --dt 0.5 --error-budget 1.0 --nlive 256 --Ebv-max 0 --injection ./nmma/output/injection.json --injection-num $macroeventID --injection-detection-limit 22,22,22 --injection-outfile ./nmma/example_files/csv_lightcurve/outdir/$macroeventID/lc.csv --generation-seed 42 --filters g,r,i  --ztf-sampling --ztf-uncertainties --plot --remove-nondetections --optimal-augmentation --optimal-augmentation-filters u,g,r,i,z,y,J,H,K --optimal-augmentation-N 100
```

Comment:
I think should be:
mkdir ./nmma/output/EMdata/$macroeventID/
light_curve_analysis --model Bu2019lm --svd-path ./nmma/svdmodels --gptype tensorflow  --outdir ./nmma/output/EMdata/outdir/$macroeventID --label injection_Bu2019lm --prior ./nmma/priors/Bu2019lm.prior --tmin 0 --tmax 7 --dt 0.5 --error-budget 1.0 --nlive 256 --Ebv-max 0 --injection ./nmma/output/injection.json --injection-num $macroeventID --injection-detection-limit 22,22,22 --injection-outfile ./nmma/output/EMdata/$macroeventID/lc.csv --generation-seed 42 --filters g,r,i  --ztf-sampling --ztf-uncertainties --plot --remove-nondetections --optimal-augmentation --optimal-augmentation-filters u,g,r,i,z,y,J,H,K --optimal-augmentation-N 100